test/local/account_routes.js

```diff
@@ -673,3 +675,110 @@ test(
     )
   }
 )
+
+test(
+  '/account/create validates metrics context data',
```

Comment:
A general discussion point rather than a comment on this PR.

I think the reason this file is growing so quickly (especially w.r.t. [that other PR I have open at the moment](https://github.com/mozilla/fxa-auth-server/pull/1259)) is that the tests are named for and focused around their respective assertions. If instead we named the tests for their setup and had one canonical test setting up each path through a route, we could add our assertions to existing test cases most of the time (and I realise I'm just as guilty of doing it the other way btw). Does that make sense?

Hopefully with the tests named that way it would be obvious to our future selves to re-use existing test setup as much as possible, but we could also add a big fat comment to the top of the module explaining the approach.

I'd be happy to do a dedicated PR that sweeps through and rejigs everything, after this and anyone else's in-flight PRs land (but before #1259 lands, I appreciate [your comment](https://github.com/mozilla/fxa-auth-server/pull/1259#issuecomment-222570698) about how long my tests are over there).